### DISCUSSION TITLE
umap parameter optimization

### DISCUSSION THREAD

**Discussion by cateto** [drob-xx/TopicTuner#8] [General]
Hello.
your application is awesome.
Are you planning to add function umap parameter optimization?

**Comment by drob-xx** [comment]
Hi @cateto, Thanks very much for taking TopicTuner for a spin - glad you like it. It is definitely on my mind. When I began the project I assumed that UMAP tuning would be on the agenda. However, what I found was that you can do a lot with HDBSCAN tuning and in some preliminary testing I didn't find playing with UMAP parameters particularly compelling. However, I would be very interested in seeing use cases where that wasn't the case. If you have a dataset you think would benefit from UMAP tuning I'd love to see it. Depending on the demand and utility I would be very interested in pursuing this.

**Reply by cateto** [reply]
okay. you're right. I operate 'n_neighbors' in umap parameter.
I saw a little differnce, too..
(Even this seems to be due to randomness of umap algorithm)
If i found that case I will share. 
Have a good day:D

**Comment by drob-xx** [comment]
Closed as resolved.
